File: connect.py
#!C:\Users\Personal\AppData\Local\Programs\Python\Python37
print("Content-Type:text/html")
print()
import cgi

print("<h1>Welcome to Python<h1>")
print("<hr/>")
print("<h1>Using input tag</h1>")
print("<body bgcolor='red'>")

form=cgi.FieldStorage()

name=form.getvalue("name")
email=form.getvalue("email")
phone=form.getvalue("phone")
project=form.getvalue("project")
message=form.getvalue("message")

import mysql.connector

con=mysql.connector.connect(user='root',password='',host='localhost',database='test')
cur=con.cursor()

cur.execute("insert into porfolio values(%s,%s,%i,%s,%s)",(name,email,phone,project,message))
con.commit()

cur.close()
con.close()
print("<h3>Message INserted Successfully</h3>")
print("<a href='index.html'> click here to go back</a>")

File: connect.php
<?php
$servername = "localhost";
$username = "root";
$password = "";
$dbname = "test";
  
// Create connection
$conn = mysqli_connect($servername, $username, $password, $dbname);
// Check connection
if (!$conn) {
  die("Connection failed: " . mysqli_connect_error());
}
$name= $_POST['name'];
$email= $_POST['email'];
$phone= $_POST['phone'];
$project= $_POST['project'];
$project= $message['message'];
  
$sql = "INSERT INTO test (name,email,phone,project,message) VALUES ('$name','$email','$phone','$project','$message')";
  
if (mysqli_query($conn, $sql)) {
  echo "New record created successfully";
} else {
  echo "Error: " . $sql . "<br>" . mysqli_error($conn);
}
  
mysqli_close($conn);
?>